### PR TITLE
bump to v7.2.0

### PR DESCRIPTION
**Pull request by yelhamer**
# Release checklist

- [x] Ensure all [milestoned issues/PRs](https://github.com/mandiant/capa/milestones) are addressed, or reassign to a new milestone.
- [ ] Add the `don't merge` label to all PRs that are close to be ready to merge (or merge them if they are ready) in [capa](https://github.com/mandiant/capa/pulls) and [capa-rules](https://github.com/mandiant/capa-rules/pulls).
- [x] Ensure the [CI workflow succeeds in master](https://github.com/mandiant/capa/actions/workflows/tests.yml?query=branch%3Amaster).
- [x] Ensure that `python scripts/lint.py rules/ --thorough` succeeds (only `missing examples` offenses are allowed in the nursery). You can [manually trigger a thorough lint](https://github.com/mandiant/capa-rules/actions/workflows/tests.yml) in CI via the "Run workflow" option. 
  - [ ] https://github.com/mandiant/capa-rules/actions/runs/10419799457
  - [x] https://github.com/mandiant/capa-rules/actions/runs/10421249999
- [x] Review changes
  - capa https://github.com/mandiant/capa/compare/\<last-release\>...master
  - capa-rules https://github.com/mandiant/capa-rules/compare/\<last-release>\...master
- [x] Update [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md)
  - Do not forget to add a nice introduction thanking contributors
  - Remember that we need a major release if we introduce breaking changes
  - Sections: see template below
  - Update `Raw diffs` links
  - Create placeholder for `master (unreleased)` section
    ```
    ## master (unreleased)

    ### New Features

    ### Breaking Changes

    ### New Rules (0)

    -

    ### Bug Fixes

    ### capa explorer IDA Pro plugin

    ### Development

    ### Raw diffs
    - [capa <release>...master](https://github.com/mandiant/capa/compare/<release>...master)
    - [capa-rules <release>...master](https://github.com/mandiant/capa-rules/compare/<release>...master)
    ```
- [x] Update [capa/version.py](https://github.com/mandiant/capa/blob/master/capa/version.py)
- [x] Create a PR with the updated [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md) and [capa/version.py](https://github.com/mandiant/capa/blob/master/capa/version.py). Copy this checklist in the PR description.
- [ ] After PR review, merge the PR and [create the release in GH](https://github.com/mandiant/capa/releases/new) using text from the [CHANGELOG.md](https://github.com/mandiant/capa/blob/master/CHANGELOG.md).
- Verify GH actions
  - [ ] [upload artifacts](https://github.com/mandiant/capa/releases)
  - [ ] [publish to PyPI](https://pypi.org/project/flare-capa)
  - [ ] [create tag in capa rules](https://github.com/mandiant/capa-rules/tags)
  - [ ] [create release in capa rules](https://github.com/mandiant/capa-rules/releases)
- [ ] [Spread the word](https://twitter.com)
- [ ] Update internal service

### CODE REVIEW

**Comment by yelhamer** [comment]
This release has the early release of the webui, which makes me think that this should be a major release? However, since it is only a small release I figured that we could release it as 7.2.0 and save version 8.0.0 for when the webui is finished (and maybe the VMRay and Bin2Export extractors are finished).

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,33 @@
...
+
+### Development
+
+### Raw diffs
+- [capa v7.2.0...master](https://github.com/mandiant/capa/compare/v7.2.0...master)
+- [capa-rules v7.2.0...master](https://github.com/mandiant/capa-rules/compare/v7.2.0...master)
+
+### v7.2.0
+The v7.2.0 release adds a web-based UI for online and offline usage, as well as a DRAKVUF sandbox feature extractor.
+Additionally, we fixed several bugs and added other features such as extracting names from dynamically resolved APIs in the IDA extractor.
```
Comment (mr-tz) [inline CHANGELOG.md:25]:
```suggestion
capa v7.2.0 adds support to analyze DRAKVUF sandbox dynamic analysis results. This release also introduces a first version of capa explorer web: a web-based user interface to inspect capa results using your browser. capa explorer web is available at https://mandiant.github.io/capa/explorer/#/.

These enhancements have been contributed by @yelhamer and @s-ff  as part of their Google Summer of Code 2024 projects.

Additionally, we fixed several bugs handling ELF files and added support to the IDA Pro extractor to leverage analyst recovered API names.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,36 @@
...
+### capa explorer IDA Pro plugin
+
+### Development
+
+### Raw diffs
+- [capa v7.2.0...master](https://github.com/mandiant/capa/compare/v7.2.0...master)
+- [capa-rules v7.2.0...master](https://github.com/mandiant/capa-rules/compare/v7.2.0...master)
+
+### v7.2.0
+capa v7.2.0 adds support to analyze DRAKVUF sandbox dynamic analysis results. This release also introduces a first version of capa explorer web: a web-based user interface to inspect capa results using your browser. capa explorer web is available at https://mandiant.github.io/capa/explorer/#/.
```
Comment (williballenthin) [inline CHANGELOG.md:24]: I think we should include links to more documentation about these new features. They are exciting! They deserve more than one sentence each, even if its not inline.

Comment (yelhamer) [reply CHANGELOG.md]: I have updated this now, please feel free if you would like to reword it.

**Review by williballenthin — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,36 @@
...
+### capa explorer IDA Pro plugin
+
+### Development
+
+### Raw diffs
+- [capa v7.2.0...master](https://github.com/mandiant/capa/compare/v7.2.0...master)
+- [capa-rules v7.2.0...master](https://github.com/mandiant/capa-rules/compare/v7.2.0...master)
+
+### v7.2.0
+capa v7.2.0 introduces a first version of capa explorer web: a web-based user interface to inspect capa results using your browser. Now, users can upload their samples or sandbox reports to the online web instance and get a list of the capabilities extracted from it. Users can also filter, sort, and see the details of each extracted capabilitiy which makes analyzing samples much easier. capa explorer web was worked on by @s-ff as part of a [GSoC project](https://summerofcode.withgoogle.com/programs/2024/projects/cR3hjbsq), and it is available at https://mandiant.github.io/capa/explorer/#/.
```
Comment (mr-tz) [inline CHANGELOG.md:24]:
```suggestion
capa v7.2.0 introduces a first version of capa explorer web: a web-based user interface to inspect capa results using your browser. Users can inspect capa result JSON documents in an online web instance or a standalone HTML page for offline usage. capa explorer supports interactive exploring of capa results to make it easier to understand them. Users can filter, sort, and see the details of all identified capabilities. capa explorer web was worked on by @s-ff as part of a [GSoC project](https://summerofcode.withgoogle.com/programs/2024/projects/cR3hjbsq), and it is available at https://mandiant.github.io/capa/explorer/#/.
```

**Comment by mr-tz** [comment]
Looks great now, thanks! Waiting on https://github.com/mandiant/capa/pull/2156 and then can merge this later today.

**Review by mr-tz — APPROVED** [review]
Thanks!

**Pull request merged**